doc/nrf/protocols/thread/tools.rst

```diff
@@ -281,13 +281,13 @@ Build the OpenThread POSIX applications by performing the following steps:
       ./script/bootstrap
 
 #. Build the applications with the required options.
-   For example, to build the ``ot-cli`` application with support for Thread v1.1, run the following command::
+   For example, to build the ``ot-cli`` application with support for Thread v1.2, run the following command::
 
-      ./script/cmake-build posix -DOT_THREAD_VERSION=1.1
+      ./script/cmake-build posix -DOT_THREAD_VERSION=1.2
 
-   Alternatively, to build the ``ot-daemon`` and ``ot-ctl`` applications with support for Thread v1.2, run the following command::
+   Alternatively, to build the ``ot-daemon`` and ``ot-ctl`` applications with support for Thread, run the following command::
```

Comment:
This is something that was here previously already, but is the change from 'ot-cli' to 'ot-ctl' intentional? Anything isn't specified in either example command.

Does support for Thread need to be mentioned if no version is specified?

Reply:
@wiba-nordic The statement that we can build thread for version 1.2 is kind of obsolete since the newest is now 1.3. We are also providing usually only libs for the newest version of thread. So instructing users to build thread for 1.2 is confusing.